marketplace_builder/views/pages/tutorials/notifications/parsing-api-response.liquid

```diff
@@ -2,14 +2,14 @@
 converter: markdown
 metadata:
   title: Parsing an API Response
-  description: This guide will help you parse an API response. In this example, you will store part of the response in a Custom Attribute for later reference using GraphQL mutations.
+  description: This guide will help you parse an API response. In this example, you will store part of the response in a Propertyfor later reference using GraphQL mutations.
 slug: tutorials/notifications/parsing-api-response
 searchable: true
 ---
 
 This guide will help you parse an API response.
 
-In this example, you will store part of the response in a Custom Attribute for later reference using GraphQL mutations in `callback`, since you are able to access response body via `response.body`.
+In this example, you will store part of the response in a Propertyfor later reference using GraphQL mutations in `callback`, since you are able to access response body via `response.body`.
```

Comment:
```suggestion
In this example, you will store part of the response in a Property for later reference using GraphQL mutations in `callback`, since you are able to access response body via `response.body`.
```